### PR TITLE
Release/1.2.2

### PR DESCRIPTION
**Pull request by QuintinWillison**

### CODE REVIEW

File: Spec/RealtimeClientConnection.swift [hunk 1]
```diff
@@ -2,7 +2,7 @@
 //  RealtimeClient.connection.swift
 //  ably
 //
-//  Created by Ricardo Pereira on 03/11/2015.
```
Comment (QuintinWillison) [inline Spec/RealtimeClientConnection.swift:5]: Oops, automation, grumble grumble, etc..

I shall push a commit up to fix this.

Comment (QuintinWillison) [reply Spec/RealtimeClientConnection.swift]: https://github.com/ably/ably-cocoa/pull/1081/commits/643d4aee4ad6c3ee57c81c7cdc9db2ea788c345c

**Review by ricardopereira — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,46 @@
...
+This means that the previous approach we supported of conforming to `ARTPushRegistererDelegate` on your `UIApplicationDelegate` no longer works.
+
+Going forward apps should use the `pushRegistererDelegate` property on client options to supply this delegate ([\#1075](https://github.com/ably/ably-cocoa/pull/1075)).
+
+***Reduced Dependencies***
+
+We've removed both KSCrash ([\#1073](https://github.com/ably/ably-cocoa/pull/1073)) and SAMKeychain ([\#1067](https://github.com/ably/ably-cocoa/pull/1067)).
+
+KSCrash, in particular, presented a heavy build burden and was causing issues for some users at runtime in respect of how it was catching unhandled exceptions and reporting them to Ably's Sentry service.
+The strategy for handling uncaught exceptions, including how to log or transmit them, should be selected and implement by the app developer, not a library dependency.
```
Comment (paddybyers) [inline CHANGELOG.md:25]: In Cocoa there is no effective way to catch selectively only those exceptions that occur within the library, and the attempt to do that using KCrash was interfering with app developers' own exception handling. Our advice is that app developers incorporate their own application-wide exception handling and crash reporting.

**Review by paddybyers — APPROVED** [review]
LGTM, but see comment

File: CHANGELOG.md (hunk 2 above)
Comment (QuintinWillison) [reply CHANGELOG.md]: https://github.com/ably/ably-cocoa/pull/1081/commits/38398b1e2ac338294174e890ca2ef0a21bcd1076

**Pull request merged**